essay.tex

```diff
@@ -27,19 +27,32 @@
 
 \section{Introduction}
 
-Write your introduction here. A brief introduction is recommended, which should outline key details of the chosen topic and the reviewed papers, motivate the work, and provide a roadmap of key points to the reader. The motivation is quite important here, as essays should have a contribution (i.e., what is the point of the essay, and what does the reader take away from the essay) and the link between the motivation (in the introduction) and the contribution (in the conclusion) should be made clear.
+In this essay I will outline the key benefits and drawbacks of daily scrum meetings and using my knowledge gained from research to discuss how they could be leveraged or overcome to make daily scrum meetings as effective as possible. I will first outline what a daily scrum meeting is and why it is important and useful, I will then discuss the benefits, drawbacks, and solutions to those drawbacks of efficient daily scrum meetings found in my research and give my own conclusions, followed by the final conclusion. In this paper I hope to provide solutions to the most common obstacles to having an efficient daily meeting and highlight good practice in a daily meetings. After reading this you should know how to make the most out of daily scrum meetings and be as productive as possible.
```

Comment:
To help *motivate* this research, you could refer to some [opinion pieces](http://dl.acm.org/citation.cfm?id=957730&CFID=697395244&CFTOKEN=53777058) in professional magazines that criticise stand-up meetings. This would better situate and justify your musings and explorations.